util/printer/workflow-printer.go

```diff
@@ -76,16 +76,48 @@ func printTable(wfList []wfv1.Workflow, out io.Writer, opts PrintOpts) {
 		}
 		_, _ = fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%d", wf.ObjectMeta.Name, WorkflowStatus(&wf), ageStr, durationStr, priority)
 		if opts.Output == "wide" {
-			pending, running, completed := countPendingRunningCompleted(&wf)
+			pending, running, completed := countPendingRunningCompletedNodes(&wf)
 			_, _ = fmt.Fprintf(w, "\t%d/%d/%d", pending, running, completed)
 			_, _ = fmt.Fprintf(w, "\t%s", parameterString(wf.Spec.Arguments.Parameters))
 		}
 		_, _ = fmt.Fprintf(w, "\n")
 	}
+
+	completed, incomplete := countCompletedWorkflows(wfList)
+	if completed > 100 || incomplete > 100 {
+		_, _ = fmt.Fprint(w, "\n")
+		fmt.Fprintf(w, "You have at least ")
```

Comment:
Should this also disregard returned values for consistency?
```suggestion
		_, _ = fmt.Fprintf(w, "You have at least ")
```